nix/cabal-licenses.nix

```diff
@@ -1,2 +1,5 @@
 stdenv: with stdenv.lib.licenses;
-{ BSD-3-Clause = bsd3; }
+{ BSD-3-Clause = bsd3;
+  BSD-2-Clause = bsd2;
+  MIT = mit;
+}
```

Comment:
@angerman Should this change be dropped, so as not to conflict with #11?

Reply:
yep. this is dead :-)

Reply:
I'll rebase #11, after this to clean up.